torch/backends/_nnapi/serializer.py

```diff
@@ -1080,13 +1080,23 @@ def _do_add_binary(self, node, opcode, fuse_code, *, qparams=None):
             scale, zp = qparams
             out_oper = out_oper._replace(scale=scale, zero_point=zp)
 
+        out_id = self.add_tensor_operand(node.outputsAt(0), out_oper)
+        for idx, (d0, d1) in enumerate(zip(in0_oper.shape, in1_oper.shape)):
+            if d0 == 1 and d1 == 0:
+                self.forward_operand_shape(out_id, idx, in1_id, idx)
+            elif d0 == 0 and d1 == 1:
+                self.forward_operand_shape(out_id, idx, in0_id, idx)
+            elif d0 == 0 and d1 == 0:
+                # FIXME need to add runtime check for equality
```

Comment:
I suggest we throw here until you're ready to implement this.

Reply:
Does this implementation work?